src/napari/_vispy/utils/zoom.py

```diff
@@ -0,0 +1,81 @@
+"""Vispy zoom box overlay."""
+
+from __future__ import annotations
+
+import numpy as np
+
+from napari.viewer import ViewerModel
+
+
+def _get_dim_info(
+    mins: np.ndarray, maxs: np.ndarray
+) -> tuple[np.ndarray, np.ndarray]:
```

Comment:
I would rename this to something more explicit (like `_get_zoombox_center_and_size`)